packages/react-ui-tracking/package.json

```diff
@@ -13,32 +13,29 @@
     "build:es": "BABEL_ENV=es babel --ignore __tests__ -d es src",
     "build:es:watch": "yarn run build:es -- --watch",
     "build:watch": "yarn run build -- --watch",
-    "test": "NODE_ENV=test jest",
+    "test": "jest",
     "test:watch": "yarn run test -- --watch",
     "lint": "eslint src",
     "lint:fix": "yarn run lint -- --fix"
   },
   "jest": {
     "setupFiles": [
-      "<rootDir>/test/shim.js",
       "<rootDir>/test/setup.js"
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
-      "/mocks/"
+      "/mocks/",
+      "./es",
+      "./lib"
     ]
   },
   "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
     "@rentpath/eslint-config-rentpath": "3.0.1",
-    "babel-cli": "6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^9.0.0",
+    "babel-core": "^7.0.0-bridge",
+    "babel-eslint": "^10.0.1",
```

Comment:
do we care to upgrade to `react 16.6.0` in here to be consistent?

Reply:
Doh! Somehow I missed this good catch!

Reply:
Fixed